### PR TITLE
Settings/multi-email: fix some bugs + improvements READY TO MERGE

### PR DESCRIPTION
**Pull request by balat**

### CODE REVIEW

**Comment by vasilisp** [comment]
Looks good, let's wait for Travis to finish.

File: template.distillery/PROJECT_NAME_settings.eliom [hunk 1]
```diff
@@ -49,42 +50,57 @@ let%shared buttons_of_email is_main_email is_validated email =
...
   let li_of_email = li_of_email main_email in
-  let%lwt li_list = Lwt_list.map_s li_of_email l in
+  let%lwt li_list = Lwt_list.map_s li_of_email emails in
   Lwt.return @@ ul li_list
 
 (* Return a list with information about emails *)
-let%client ul_of_emails =
-  ~%(Eliom_client.server_function [%derive.json : int64] ul_of_emails)
+let%server get_emails () =
+  let open Eliom_content.Html.F in
```
Comment (dannywillems) [inline template.distillery/PROJECT_NAME_settings.eliom]: Unused opened module :).

Comment (balat) [inline template.distillery/PROJECT_NAME_settings.eliom]: Fixed. Thanks

File: template.distillery/PROJECT_NAME_settings.eliom [hunk 2]
```diff
@@ -49,42 +50,57 @@ let%shared buttons_of_email is_main_email is_validated email =
...
-  let%lwt l = Os_db.User.emails_of_userid userid in
   let li_of_email = li_of_email main_email in
-  let%lwt li_list = Lwt_list.map_s li_of_email l in
+  let%lwt li_list = Lwt_list.map_s li_of_email emails in
   Lwt.return @@ ul li_list
 
 (* Return a list with information about emails *)
-let%client ul_of_emails =
-  ~%(Eliom_client.server_function [%derive.json : int64] ul_of_emails)
+let%server get_emails () =
```
Comment (dannywillems) [inline template.distillery/PROJECT_NAME_settings.eliom]: Maybe we can add `val get_emails : unit -> string list` in `Os_current_user`.

**Comment by dannywillems** [comment]
Ok also for me.

**Pull request merged**